components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -107,14 +143,22 @@ const LeagueEntries = ({
           data-testid="league-entry-pick-button-container"
         >
           {!isEliminated && (
-            <Link href={linkUrl} data-testid="league-entry-pick-button-link">
-              <Button
-                className="league-entry-pick-button"
-                data-testid="league-entry-pick-button"
-                label={isPickSet ? 'Change Pick' : 'Make Pick'}
-                variant={isPickSet ? 'secondary' : 'default'}
-              />
-            </Link>
+            <LinkCustom
+              aria-disabled={isLockedOut === true ? 'true' : 'false'}
```

Comment:
Q: Why can't we use truthy/falsy values here and write the ternary operation like the following: `isLockedOut ? 'true' : 'false'`

Reply:
@ryandotfurrer could you help answer this? i didnt understand it as well

Reply:
Agree. isLockedOut is a truthy value on its own. It's unnecessary to include the isLockedOut === true part. 
aria-disabled={isLockedOut ? 'true' : 'false'}  is cleaner and functions the same